lib/rules/lint-simple-unless.js

```diff
@@ -142,6 +147,14 @@ module.exports = class LintSimpleUnless extends Rule {
             this._logMessage(message, loc.line, loc.column, actual);
           }
 
+          if (blacklist.length > 0 && blacklist.indexOf(param.path.original) > -1) { // blacklist.includes(param.path.original)
```

Comment:
I think we can use `.includes` now (we dropped Node 4 support a little while back),  mind swapping to that?

Reply:
Sounds good! Also updated a similar usage in the whitelist check above.